--------------------
q2 > Test #3
Test #3
ERROR
with-limit: out of time
  context...:
   /proj/racket/plt-released/recent/share/pkgs/sandbox-lib/racket/sandbox.rkt:379:0: call-with-limits
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/result.rkt:99:3
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:28:2
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:85:13: the-tests
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:60:0: apply-test-suite
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/text-ui.rkt:243:0: run-tests14
   /home/jmk/src/CODECUT/set07/pdp-manishyadav-williamwan/set07/q2-bb.rkt: [running body]

--------------------
--------------------
q2 > Test #5
Test #5
ERROR
with-limit: out of memory
  context...:
   /proj/racket/plt-released/recent/share/pkgs/sandbox-lib/racket/sandbox.rkt:379:0: call-with-limits
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/result.rkt:99:3
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:28:2
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:85:13: the-tests
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:60:0: apply-test-suite
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/text-ui.rkt:243:0: run-tests14
   /home/jmk/src/CODECUT/set07/pdp-manishyadav-williamwan/set07/q2-bb.rkt: [running body]

--------------------
--------------------
q2 > Test #10
Test #10
ERROR
with-limit: out of time
  context...:
   /proj/racket/plt-released/recent/share/pkgs/sandbox-lib/racket/sandbox.rkt:379:0: call-with-limits
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/result.rkt:99:3
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:28:2
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:85:13: the-tests
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:60:0: apply-test-suite
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/text-ui.rkt:243:0: run-tests14
   /home/jmk/src/CODECUT/set07/pdp-manishyadav-williamwan/set07/q2-bb.rkt: [running body]

--------------------
--------------------
q2 > Test #11
Test #11
ERROR
with-limit: out of time
  context...:
   /proj/racket/plt-released/recent/share/pkgs/sandbox-lib/racket/sandbox.rkt:379:0: call-with-limits
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/result.rkt:99:3
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:28:2
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:85:13: the-tests
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:60:0: apply-test-suite
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/text-ui.rkt:243:0: run-tests14
   /home/jmk/src/CODECUT/set07/pdp-manishyadav-williamwan/set07/q2-bb.rkt: [running body]

--------------------
--------------------
q2 > Test #12
Test #12
ERROR
with-limit: out of time
  context...:
   /proj/racket/plt-released/recent/share/pkgs/sandbox-lib/racket/sandbox.rkt:379:0: call-with-limits
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/result.rkt:99:3
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:28:2
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:85:13: the-tests
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:60:0: apply-test-suite
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/text-ui.rkt:243:0: run-tests14
   /home/jmk/src/CODECUT/set07/pdp-manishyadav-williamwan/set07/q2-bb.rkt: [running body]

--------------------
--------------------
q2 > Test #13
Test #13
ERROR
with-limit: out of time
  context...:
   /proj/racket/plt-released/recent/share/pkgs/sandbox-lib/racket/sandbox.rkt:379:0: call-with-limits
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/result.rkt:99:3
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:28:2
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:85:13: the-tests
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:60:0: apply-test-suite
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/text-ui.rkt:243:0: run-tests14
   /home/jmk/src/CODECUT/set07/pdp-manishyadav-williamwan/set07/q2-bb.rkt: [running body]

--------------------
--------------------
q2 > Test #14
Test #14
ERROR
with-limit: out of time
  context...:
   /proj/racket/plt-released/recent/share/pkgs/sandbox-lib/racket/sandbox.rkt:379:0: call-with-limits
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/result.rkt:99:3
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:28:2
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:85:13: the-tests
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/private/test-suite.rkt:60:0: apply-test-suite
   /proj/racket/plt-released/recent/share/pkgs/rackunit-lib/rackunit/text-ui.rkt:243:0: run-tests14
   /home/jmk/src/CODECUT/set07/pdp-manishyadav-williamwan/set07/q2-bb.rkt: [running body]

--------------------
8 success(es) 0 failure(s) 7 error(s) 15 test(s) run
extras.rkt Wed Sep 14 08:52:19 2016
Running tests from flight.rkt...
All tests passed (16 tests).
q2.rkt appears to be in a correctly named folder. Running tests...
Running tests from q2.rkt...
All tests passed (13 tests).
7
